CrewChiefV4/Events/OvertakingAidsMonitor.cs

```diff
@@ -62,20 +64,28 @@ override protected void triggerInternal(GameStateData previousGameState, GameSta
                 }
                 if (currentGameState.SessionData.IsNewLap)
                 {
-                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap)
+                    if (drsAvailableOnThisLap && !hasUsedDrsOnThisLap
+                        || (CrewChief.gameDefinition.gameEnum == GameEnum.RF2_64BIT && drsAvailableInAnyZoneOnThisLap && !hasUsedDrsOnThisLap))
                     {
                         audioPlayer.playMessage(new QueuedMessage("missed_available_drs", MessageContents(folderDontForgetDRS), 0, this));
                     }
                     drsAvailableOnThisLap = currentGameState.OvertakingAids.DrsAvailable;
+                    drsAvailableInAnyZoneOnThisLap = false;
```

Comment:
is this right?

Reply:
I will change this, but I will keep code below, because as far as I understand so far, DRS may become available for use in multiple areas of the lap.  I in fact wonder, maybe we don't even need to special case rF2 vs R3E and use same variable for both of them.   On the other hand, it is less risky.

Reply:
Do you happen to remember what exactly DrsAvailable == true in R3E means?  It means user can enable it at start/finish line crossing?

Reply:
yes, the user can use DRS on the current lap if he wants to

Reply:
Can it ever become true later in the lap?